src/verifier.rs

```diff
@@ -66,6 +66,10 @@ impl ServerVerifier {
         self.last_result.load(Ordering::Acquire)
     }
 
+    pub fn update_last_result(&self, v: i64) {
+        self.last_result.store(v, Ordering::Relaxed);
```

Comment:
I _think_ `Relaxed` is the right choice here, but I admit to not being 100% sure. (Ditto L210).

Calling this out as something I'd appreciate input on.